### DISCUSSION TITLE
Hvorfor dukker Mette Hornbek Hansen ikke op i frivilligsøgningen hos Kulturens?

### DISCUSSION THREAD

**Discussion by MartinQwist** [xooom-community/public#12] [Questions] [answered]
Jeg prøver at søge efter Mette Hornbek Hansen i søgningen, hvor jeg skriver "Mette". Men hun dukker ikke op.

![image](https://github.com/user-attachments/assets/b92c7efc-fa9f-48b3-87d2-a880b079cdd8)

![image](https://github.com/user-attachments/assets/520af52c-ccbf-47b6-93c7-03d90583da2d)

**Comment by MartinQwist** [comment]
Og lidt bonusinfo fra samme bruger: 

Hun er administrator for to forskellige organisationer: Sydhavn Teater og Teater GLIMT. Hun har oprettet sin bruger med sin private gmail (mehornbek@gmail.com). Den vil hun gerne skifte til sin mail for Sydhavn Teater (mette@sydhavnteater.dk). Men når hun gør det, så får hun besked om, at email adressen allerede er i brug. 

Men jeg kan ikke søge mailadressen mette@sydhavnteater.dk frem.

(I det lidt støtte billede skal vi kigge på, hvad en arrangør gør og oplever, når de skal gå på tværs af organisationer indenfor samme frivilligorg. og på tværs af frivilligorganisationer. De to typer cases driller ofte på forskellige måder....)

**Reply by antonlund18** [reply]
Det ligner det er en konsekvens, fra da jeg ændrede det, så man ikke kan se oplysninger for administratorer, før de har accepteret invitationen. Her ændrede jeg det også, så de ikke indgik i søgningen, før de havde accepteret invitationen (så man fx ikke kan søge på deres navn, og få deres informationer den vej igennem).

Her ligner det også, at Mette ikke har aktiveret hendes frivilligprofil hos jer endnu, og hun bliver derfor ikke inkluderet i søgningen. Der er nok en opgave i, at finde ud af hvordan vi håndterer det. Jeg tror den eneste måde, at man kan blive frivillig på, er ved at oprette sig hos frivilligorganisationen. I så fald, burde det vel være okay at inkludere deres informationer, selvom de ikke er aktiveret?

Men uanset hvad, så er mehornbek@gmail.com og mette@sydhavnteater.dk to forskellige brugere, og hun vil ikke kunne få lov til at skifte mail på mehornbek@gmail.com, så længe at mette@sydhavnteater.dk eksisterer. Hun bør slette hendes bruger med mehornbek@gmail.com, og logge ind med mette@sydhavnteater.dk på jeres portal.

**Comment by MartinQwist** [comment]
Hvordan løser vi dette, så brugere med arrangørrettigheder fremgår af frivilligsøgningen? Som jeg forstår det er det ved at acceptere en invitation - men det forstår jeg ikke helt. F.eks. dukker hverken jeg eller Emma op på søgningen (martin@kulturens.dk, Emma@kulturens.dk), hvilket passer med, at vi også er "arrangører" i Kulturens Frivillige. Men vi har ikke så vidt vi ved modtaget en invitation...

**Reply by antonlund18** [reply]
Det er egentligt to forskellige problemstillinger.

1. Nogle brugere er blevet oprettet som frivillige, men deres brugere er ikke blevet aktiveret, og de ligger derved i frivilligoversigten, men kommer ikke frem når man søger på dem. Disse brugere burde faktisk måske slet ikke fremgå i frivilligoversigten? Jeg tror det er hvis de frivillige bliver inviteret til at blive admins og nedgraderet til frivillige, før de når at acceptere invitationen, men det burde slet ikke være muligt mere.

2. Brugere med arrangørrettigheder fremgår ikke i frivilligoversigten. Dette er et bevidst valg, der er blevet lavet, og kan meget nemt ændres, hvis det er det der ønskes.

**Reply by antonlund18** [reply, marked as answer]
@MartinQwist Jeg har forbedret søgningen en smule, og også løst problem `#1`. Det er kun i preview lige nu. Du må gerne sige til, hvis du stadig oplever problemer med søgningen, eller hvis du ønsker at inkludere arrangører i frivilligoversigten.

**Reply by MartinQwist** [reply]
Det lyder godt. Jeg går lige tilbage til Rethinker og beder dem teste samt at tage stilling til, om arrangører skal med på frivilliglisten.
